lib/OpenQA/CacheService/Model/Cache.pm

```diff
@@ -142,7 +145,8 @@ sub track_asset {
     eval {
         my $db  = $self->sqlite->db;
         my $tx  = $db->begin('exclusive');
-        my $sql = "INSERT OR IGNORE INTO assets (filename, size, last_use) VALUES (?, 0, strftime('%s','now'))";
+        my $sql = "INSERT INTO assets (filename, size, last_use) VALUES (?, 0, strftime('%s','now'))"
+          . "ON CONFLICT (filename) DO UPDATE SET pending=1";
```

Comment:
do we need a space between `'now'))` and `ON`?

Reply:
We don't need a space and I didn't change the formatting compared to how it was before.